docs/sidebar.json

```
@@ -61,8 +61,6 @@
     "reference/html-forms",
     "reference/api",
     {
-      "type": "category",
-      "label": "Command Line Interface (CLI)",
       "Command Line Interface (CLI)": [
```

This won't fix the docs. Revert the changes and:

```suggestion
      "items": [
```

---

Why would it not?

https://docusaurus.io/docs/sidebar#creating-a-hierarchy

> Note: it's possible to use the shorthand syntax to create nested categories:
> ```
> module.exports = {
>   docs: {
>     Guides: [
>       'creating-pages',
>       {
>         Docs: [
>           'docs-introduction',
>           'docs-sidebar',
>           'markdown-features',
>           'versioning',
>         ],
>       },
>     ],
>   },
> };
> ```

---

Looks like I learned something new ;)